flink-ml-lib/src/main/java/org/apache/flink/ml/feature/maxabsscaler/MaxAbsScalerModel.java

```diff
@@ -0,0 +1,162 @@
+        RowTypeInfo outputTypeInfo =
+                new RowTypeInfo(
+                        ArrayUtils.addAll(
+                                inputTypeInfo.getFieldTypes(), TypeInformation.of(Vector.class)),
+                        ArrayUtils.addAll(inputTypeInfo.getFieldNames(), getOutputCol()));
+        DataStream<Row> output =
+                BroadcastUtils.withBroadcastStream(
+                        Collections.singletonList(data),
+                        Collections.singletonMap(broadcastModelKey, maxAbsScalerModel),
+                        inputList -> {
+                            DataStream input = inputList.get(0);
+                            return input.map(
+                                    new PredictOutputFunction(broadcastModelKey, getInputCol()),
+                                    outputTypeInfo);
+                        });
+        return new Table[] {tEnv.fromDataStream(output)};
+    }
+
+    @Override
+    public Map<Param<?>, Object> getParamMap() {
+        return paramMap;
+    }
+
+    @Override
+    public void save(String path) throws IOException {
+        ReadWriteUtils.saveMetadata(this, path);
+        ReadWriteUtils.saveModelData(
+                MaxAbsScalerModelData.getModelDataStream(modelDataTable),
+                path,
+                new MaxAbsScalerModelData.ModelDataEncoder());
+    }
+
+    /**
+     * Loads model data from path.
+     *
+     * @param tEnv Stream table environment.
+     * @param path Model path.
+     * @return MaxAbsScalerModel model.
+     */
+    public static MaxAbsScalerModel load(StreamTableEnvironment tEnv, String path)
+            throws IOException {
+        MaxAbsScalerModel model = ReadWriteUtils.loadStageParam(path);
+        Table modelDataTable =
+                ReadWriteUtils.loadModelData(
+                        tEnv, path, new MaxAbsScalerModelData.ModelDataDecoder());
+        return model.setModelData(modelDataTable);
+    }
+
+    /** This operator loads model data and predicts result. */
+    private static class PredictOutputFunction extends RichMapFunction<Row, Row> {
+        private final String inputCol;
+        private final String broadcastKey;
+        private DenseVector scaleVector;
+
+        public PredictOutputFunction(String broadcastKey, String inputCol) {
+            this.broadcastKey = broadcastKey;
+            this.inputCol = inputCol;
+        }
+
+        @Override
+        public Row map(Row row) {
+            if (scaleVector == null) {
+                MaxAbsScalerModelData maxAbsScalerModelData =
+                        (MaxAbsScalerModelData)
+                                getRuntimeContext().getBroadcastVariable(broadcastKey).get(0);
+                scaleVector = maxAbsScalerModelData.maxVector;
+                for (int i = 0; i < scaleVector.size(); ++i) {
+                    if (scaleVector.values[i] != 0) {
+                        scaleVector.values[i] = 1.0 / Math.abs(scaleVector.values[i]);
```

Comment:
Spark's implementation is similar to `1.0 / scaleVector.values[i]`, instead of `1.0 / Math.abs(scaleVector.values[i])`. Would it be better to follow Spark's practice?